rma/tests/test_rma.py

```diff
@@ -157,7 +162,9 @@ def test_post_init_hook(self):
 
     def test_rma_replace_pick_ship(self):
         self.warehouse.write({"delivery_steps": "pick_ship"})
-        rma = self._create_rma(self.partner, self.product, 1, self.rma_loc)
+        rma = self._create_rma(
+            self.partner, self.product, 1, self.rma_loc, self.operation
```

Comment:
Can we avoid to pass on each call the operation and change the method `_create_rma` to take `self.operation` if the value is `None`? It's for reducing all the diff in the test that makes it difficult to follow.

Reply:
Make sense

Reply:
Done